packages/core/test/unit/process/4-mutation-test-executor.spec.ts

```diff
@@ -383,4 +383,18 @@ describe(MutationTestExecutor.name, () => {
     // Assert
     expect(testInjector.logger.info).calledWithExactly('Done in %s.', '2 seconds, tops!');
   });
+
+  it('should short circuit when dryRunOnly is enabled', async () => {
+    // Arrange
+    testInjector.options.dryRunOnly = true;
```

Comment:
We probably also want to use `arrangeScenario()` here, to make sure it isn't short-circuited because there weren't any mutants.

Reply:
I think that does not really matter for this test, as we try to short circuit even before that check.
Or at least that is what is happening now. If we do want to include the earlyResults with the mutantPlan, that that needs to change